### PR TITLE
IGNITE-12351 Append additional cp tracking activity - pages sort.

### PR DESCRIPTION
**Pull request by vmalin**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/pagemem/FullPageId.java [hunk 1]
```diff
@@ -124,6 +124,13 @@ public long pageId() {
         return pageId;
     }
 
+    /**
+     * @return Effective page ID.
```
Comment (zstan) [inline modules/core/src/main/java/org/apache/ignite/internal/pagemem/FullPageId.java]: u can use 1 line comment here.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java [hunk 2]
```diff
@@ -163,6 +173,20 @@ public void onWalCpRecordFsyncStart() {
         walCpRecordFsyncStart = System.currentTimeMillis();
     }
 
+    /**
+     *
+     */
```
Comment (zstan) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java]: empty multi linear comments are inapplicable.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java [hunk 3]
```diff
@@ -163,6 +173,20 @@ public void onWalCpRecordFsyncStart() {
...
+    public void onSplitAndSortCpPagesStart() {
+        splitAndSortCpPagesStart = System.currentTimeMillis();
+    }
+
+    /**
+     *
```
Comment (zstan) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java]: the same.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java (hunk 2 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java (hunk 3 above)
Comment (vmalin) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/CheckpointMetricsTracker.java]: fixed

**Pull request closed without merging**